tensorrt_llm/_torch/pyexecutor/kv_cache_events.py

```
@@ -0,0 +1,618 @@
+                try:
+                    payload = encoder.encode(event)
+                    self._pub.send_multipart(
+                        (
+                            self._topic_bytes,
+                            seq.to_bytes(8, "big"),
+                            payload,
+                        )
+                    )
+                    self._buffer.append((seq, payload))
+                    self.published_batches += 1
+                except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)